app-connector-client/test/test_integration.js

```diff
@@ -1,14 +1,35 @@
 
 var request = require('supertest');
-var server = require("../server/app")("test/varkes_config.json");
+const mock = require('../server/app')
+const express = require("express")
 const fs = require("fs")
 const path = require("path")
+const kyma = require("@varkes/example-kyma-mock")
 const serviceMetadata = path.resolve("test/service-metadata.json")
+require("../server/keys").generatePrivateKey()
 
-if (process.env.NODE_ENV == "test") {
-    require("../server/keys").generatePrivateKey()
+const port = 10005 //! listen in different port
+const tokenURL = `http://localhost:${port}/connector/v1/applications/signingRequests/info?token=123`
+
+describe("should work", () => {
+    var kymaServer
+    before(() => { //* start kyma mock before tests
+        kyma.then(app => {
+            kymaServer = app.listen(port)
+        })
+    })
+    after(() => { //* stop kyma mock after tests
+        kymaServer.close()
+    })
 
     describe('basic routes', function () {
+        before(async () => {

```

Comment:
PLease check how we started the mock in the old test.js, no need to change the approach

Reply:
The before approach is more understandable, it's better to change others to before approach.